server/src/main/java/io/druid/client/DirectDruidClient.java

```diff
@@ -163,7 +177,9 @@ public int getNumOpenConnections()
         public ClientResponse<InputStream> handleResponse(HttpResponse response)
         {
           log.debug("Initial response from url[%s]", url);
-          startTime = System.currentTimeMillis();
+          responseStartTime = System.currentTimeMillis();
+          emitter.emit(builder.build("query/node/ttfb", responseStartTime - requestStartTime));
```

Comment:
`ttfb` stands for "time to first byte". I could spell it out, or does someone has a better suggestion?